models/User.php

```diff
@@ -990,6 +990,28 @@ public function getKeyBindings()
         return $this->keyBindings ? $this->keyBindings : self::getDefaultKeyBindings();
     }
 
+    /**
+     * @param array|null $bindings
+     * @return array|null
+     */
+    public static function strictKeybinds($bindings)
+    {
+        if($bindings == null){
+            return null;
+        }
```

Comment:
I don't think we need to support the `null` case here.

```suggestion
    public static function strictKeybinds(array $bindings): array
    {
```